sdcm/utils/raft/__init__.py

```diff
@@ -122,6 +126,14 @@ def __init__(self, node: "BaseNode") -> None:
     def is_enabled(self) -> bool:
         return True
 
+    @property
+    def consistent_topology_changes_enabled(self) -> bool:
+        with self._node.remote_scylla_yaml() as scylla_yaml:
+            if self._node.is_kubernetes():
+                return "consistent-topology-changes" in scylla_yaml.get("experimental_features", [])
```

Comment:
@vponomaryov why it is necessary to do such exceptions for k8s? can we make it seamless like on vm based nodes so we don't have to remember to have different logic for k8s?

Reply:
K8S has the `scylla_yaml` as dict. VMs have it as data class instance with attributes.
Why? Because the data class instance is not suitable for K8S where Scylla configuration is partially configured only by operator.
There are several solutions. One of them is to add `__getitem__` support for the data class instance used for VMs. So, after it we could use 'get' method everywhere.